### ISSUE TITLE
Plugin architecture

### ISSUE THREAD

**Issue by danwdart** [dadi/publish#395]
Place to discuss the plugin architecture

Based on: https://www.netlify.com/blog/2017/06/20/extending-netlify-cms-part-one-custom-widgets/

[todo]

**Comment by eduardoboucas** [comment]
Related:

- https://github.com/dadi/publish/issues/299
- https://github.com/dadi/publish/issues/313
- https://github.com/dadi/publish/issues/317

**Comment by danwdart** [comment]
Is this considered completed by the above 3 tickets being closed / dupes?

**Comment by eduardoboucas** [comment]
No, this is the main ticket for the plugin architecture. The 3 tickets above were referenced because similar discussions took place there and might contain useful information.

No movement has been made on the plugin architecture, as far as I know.

**Comment by danwdart** [comment]
Aha got you
